index.bs

```diff
@@ -1370,6 +1373,85 @@ object |transport|, run these steps.
 
 </div>
 
+# `WebTransportError` Interface #  {#web-transport-error-interface}
+
+<dfn interface>WebTransportError</dfn> is a subclass of {{DOMException}} that represents
+
+ - An error coming from the server or the network, or
+ - A reason for a client-initiated abort operation.
+
+<pre class="idl">
+[Exposed=(Window,Worker), SecureContext]
+interface WebTransportError : DOMException {
+  constructor(optional WebTransportErrorInit init = {});
+
+  readonly attribute octet? applicationProtocolCode;
+};
+
+dictionary WebTransportErrorInit {
+  octet applicationProtocolCode;
+  DOMString message;
+};
+</pre>
+
+## Internal slots  ## {#web-transport-error-internal-slots}
+
+A {{WebTransportError}} has the following internal slots.
+
+<table class="data" dfn-for="WebTransportError">
+ <thead>
+  <tr>
+   <th>Internal Slot
+   <th>Description (<em>non-normative</em>)
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><dfn>\[[ApplicationProtocolCode]]</dfn>
+   <td class="non-normative">The application protocol error code for this error, or null.
```

Comment:
The spec has two previous mentions of "application protocol code", with language referring to [QUIC] — which is missing here — but I had to search the document for them.

It would be nice to link them together, and point them to a new definition under concepts.